src/MAUI/Maui.Samples/Samples/Map/ManageOperationalLayers/ManageOperationalLayers.xaml

```diff
@@ -35,31 +32,29 @@
     <Grid Style="{DynamicResource EsriSampleContainer}">
         <esriUI:MapView x:Name="MyMapView" Style="{DynamicResource EsriSampleGeoView}" />
         <Border Style="{DynamicResource EsriSampleControlPanel}">
-            <ScrollView>
+            <ScrollView MaximumHeightRequest="350">
                 <VerticalStackLayout Padding="5" Spacing="5">
-                    <Label HeightRequest="30" Text="Use the buttons to re-arrange layers." />
+                    <Label Text="Use the buttons to re-arrange layers." />
                     <Label FontAttributes="Bold"
-                           HeightRequest="30"
                            HorizontalTextAlignment="Center"
                            Text="Layers in map" />
                     <!--
                         Item template defined in UserControl.Resources above specifies that each listbox
                         item's content consists of a label with the layer's Name and three buttons.
                     -->
                     <ListView x:Name="IncludedListView"
```

Comment:
Text wrapping splits words across lines, might need a minimum width for the label in each list view item.